app/src/main/java/de/westnordost/streetcomplete/about/CreditsFragment.kt

```diff
@@ -120,3 +125,27 @@ private fun withLinkToGithubAccount(contributor: String): String {
         "$name (<a href=\"https://github.com/$githubName\">$githubName</a>)"
```

Comment:
@westnordost I took the liberty to implement and commit this right now, since I have also asked myself the same thing during my review. See [`456fab3`](https://github.com/streetcomplete/StreetComplete/pull/3850/commits/456fab37a20011ffc92a1d8646ebfb27a28f818b).